### PR TITLE
Ui additions

### PR DESCRIPTION
**Pull request by Frueh**

### CODE REVIEW

File: raspberry-uc-system/RaspberryUserControlSystem/RaspberryBackend/Components/RaspberryPi/HIcontrol.cs [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
+            {"High Power Right", 8.660},
+            {"High Power Left", 11.000},
+            {"Defective", 133.700},
+            {"No Receiver", 200.0},
+        };
+
+        /// <summary>
         /// Sets the DACVoltage output in channel 1 to a desired voltage
         /// </summary>
         /// <param name="voltage"></param>
```
Comment (Tango1266) [inline raspberry-uc-system/RaspberryUserControlSystem/RaspberryBackend/Components/RaspberryPi/HIcontrol.cs]: paramter range could be added

File: raspberry-uc-system/RaspberryUserControlSystem/RaspberryBackend/Components/RaspberryPi/HIcontrol.cs [hunk 2]
```diff
@@ -30,5 +51,81 @@ public void connectPins(int xPin, int yPin)
...
+        public string getAudioShoeStatus()
+        {
+            return this.readPin(this.audioShoe_Pin);
+        }
+
+        /// <summary>
+        /// Return current status of tele-coil. Note that readPin returns the last written value, which does not neccessarily
+        /// represent the real status of a ping.
+        /// </summary>
+        /// <returns></returns>
```
Comment (Tango1266) [inline raspberry-uc-system/RaspberryUserControlSystem/RaspberryBackend/Components/RaspberryPi/HIcontrol.cs]: For what is it returning a string?

File: raspberry-uc-system/RaspberryUserControlSystem/RaspberryBackend/Components/SPI/ADCDAC.cs [hunk 3]
```diff
@@ -74,15 +77,44 @@ public void setDACVoltage(double voltage)
...
+                    currentDACVoltage1 = voltage;
+                }
+                else if (channel == CHANNEL_2)
+                {
+                    currentDACVoltage2 = voltage;
+                }
             }
+        }
+
+        public double getDACVoltage1()
```
Comment (Tango1266) [inline raspberry-uc-system/RaspberryUserControlSystem/RaspberryBackend/Components/SPI/ADCDAC.cs]: public method without docu

File: raspberry-uc-system/RaspberryUserControlSystem/RaspberryBackend/Components/SPI/ADCDAC.cs [hunk 4]
```diff
@@ -74,15 +77,44 @@ public void setDACVoltage(double voltage)
...
+        {
+            return currentDACVoltage1;
+        }
+        public double getDACVoltage2()
```
Comment (Tango1266) [inline raspberry-uc-system/RaspberryUserControlSystem/RaspberryBackend/Components/SPI/ADCDAC.cs]: public method without docu

File: raspberry-uc-system/RaspberryUserControlSystem/RaspberryBackend/Components/SPI/ADCDAC.cs [hunk 5]
```diff
@@ -74,15 +77,44 @@ public void setDACVoltage(double voltage)
...
+        {
+            return currentDACVoltage2;
+        }
 
-            currentDACVoltage = voltage;
+        /// <summary>
+        /// Wrapper around setDACVoltage so set channels without knowing their addresss
+        /// </summary>
+        /// <param name="voltage"></param>
```
Comment (Tango1266) [inline raspberry-uc-system/RaspberryUserControlSystem/RaspberryBackend/Components/SPI/ADCDAC.cs]: paramteter range <see also setDACVoltage2

File: raspberry-uc-system/RaspberryUserControlSystem/RaspberryBackend/RaspberryPiRemoteAPI/EndlessVC.cs [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RaspberryBackend
+{
+    /// <summary>
+    /// This class represents a Command. It it can be used to set endless volume-control.
+    /// </summary>
+    public partial class RaspberryPi
```
Comment (Tango1266) [inline raspberry-uc-system/RaspberryUserControlSystem/RaspberryBackend/RaspberryPiRemoteAPI/EndlessVC.cs]: method parameter in this class are not documented

File: raspberry-uc-system/RaspberryUserControlSystem/RaspberryBackend/RaspberryPiRemoteAPI/LightLED.cs [hunk 7]
```diff
@@ -14,10 +14,11 @@ public partial class RaspberryPi
         private const UInt16 GPIO_PIN_ID = 24;
 
         /// <summary>
-        ///  executes the Command LightLED in dependency of the parsed parameter 
+        ///  executes the Command LightLED in dependency of the parsed parameter
```
Comment (Tango1266) [inline raspberry-uc-system/RaspberryUserControlSystem/RaspberryBackend/RaspberryPiRemoteAPI/LightLED.cs]: summary starts with small letter

File: raspberry-uc-system/RaspberryUserControlSystem/TestmachineFrontend/MainWindowPartialClasses/UpdateMehods.cs [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+        /// <summary>
+        /// Not yet implemented
+        /// </summary>
+        /// <param name="result">The responded result from Backend</param>
+        public void updateGUI_PressRockerSwitch(Result result)
+        {
+            addMessage("Update", "PressRockerSwitch GUI notification not yet implemented!");
+        }
+
+        public void updateGUI_EndlessVCDown(Result result)
```
Comment (Tango1266) [inline raspberry-uc-system/RaspberryUserControlSystem/TestmachineFrontend/MainWindowPartialClasses/UpdateMehods.cs]: method not documented; see also next method.

File: raspberry-uc-system/RaspberryUserControlSystem/TestmachineFrontendTests/UpdateGUI_Tests.cs [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Windows.Media;
+using System.Windows.Shapes;
+using TestmachineFrontend;
+
+namespace TestmachineFrontendTests
```
Comment (Tango1266) [inline raspberry-uc-system/RaspberryUserControlSystem/TestmachineFrontendTests/UpdateGUI_Tests.cs]: tests not documented. Do we do that?

**Comment by Marco21718** [comment]
Nice to see many features implemented. I merge this branch because it is very important for tomorrow to show the changes in the sprint review.

**Pull request merged**